closed/OpenJ9.gmk

```diff
@@ -28,13 +28,13 @@ ifeq (,$(BUILD_ID))
   BUILD_ID := 000000
 endif
 
-# find the closest tag name
-OPENJ9_TAG    := $(shell git -C $(OPENJ9_TOPDIR)    describe --abbrev=0)
 OPENJ9_SHA    := $(shell git -C $(OPENJ9_TOPDIR)    rev-parse --short HEAD)
 OPENJ9OMR_SHA := $(shell git -C $(OPENJ9OMR_TOPDIR) rev-parse --short HEAD)
 ifeq (,$(OPENJ9_SHA))
   $(error Could not determine OpenJ9 SHA)
 endif
+# find the tag associated with OPENJ9_SHA
+OPENJ9_TAG    := $(shell git -C $(OPENJ9_TOPDIR)    describe --exact-match $(OPENJ9_SHA))
```

Comment:
This should redirect stderr to /dev/null to suppress errors (fatal: no tag exactly matches) for the common case where there are no (matching) tags. I don't think it hurts, but OPENJ9_SHA need not be used.

Also the extra spaces no longer serve any purpose now that the line has moved away from the definition of OPENJ9_SHA.